agent-control/src/agent_control.rs

```diff
@@ -323,6 +326,13 @@ where
                                 AgentControlInternalEvent::HealthUpdated(health) => {
                                     self.report_health(health);
                                 },
+                                AgentControlInternalEvent::AgentControlCdVersionUpdated(cd_version) => {
+                                    let _ = set_agent_description_version(
+                                        cd_version,
+                                        self.opamp_client.as_ref()
+                                    )
+                                    .inspect_err(|e| error!(error = %e, select_arm = "agent_control_internal_consumer", "processing version message"));
```

Comment:
Are we finally setting the agent-contro-cd version as an identifying attribute? If I got it right, `set_agent_description_version` is always updating identifying attributes 🤔